packages/ra-ui-materialui/src/button/CreateButton.js

```diff
@@ -4,18 +4,17 @@ import onlyUpdateForKeys from 'recompose/onlyUpdateForKeys';
 import {
     Fab,
     createStyles,
-    withStyles,
+    makeStyles,
     useMediaQuery,
 } from '@material-ui/core';
 import ContentAdd from '@material-ui/icons/Add';
-import compose from 'recompose/compose';
 import classnames from 'classnames';
 import { Link } from 'react-router-dom';
 import { useTranslate } from 'ra-core';
 
 import Button from './Button';
 
-const styles = theme =>
+const useStyles = makeStyles(theme =>
     createStyles({
```

Comment:
you don't need `createStyles` when using `makeStyles`

Reply:
Thanks! Will make the changes.

Reply:
I've removed the un-needed `createStyles`